CefSharp.WinForms/ChromiumWebBrowser.cs

```diff
@@ -431,79 +451,92 @@ private void InitializeFieldsAndCefIfRequired()
         }
 
         /// <summary>
-        /// Releases the unmanaged resources used by the <see cref="T:System.Windows.Forms.Control" /> and its child controls and optionally releases the managed resources.
+        /// Finalizes an instance of the <see cref="ChromiumWebBrowser"/> class.
         /// </summary>
-        /// <param name="disposing">true to release both managed and unmanaged resources; false to release only unmanaged resources.</param>
-        protected override void Dispose(bool disposing)
+        ~ChromiumWebBrowser()
+        {
+            Dispose(false);
+        }
+
+        /// <summary>
+        /// Releases all resources used by the <see cref="ChromiumWebBrowser"/> object
+        /// </summary>
+        public new void Dispose()
```

Comment:
What exactly is the reason for newing over the existing implementation? 

https://referencesource.microsoft.com/#System/compmod/system/componentmodel/Component.cs,118

Reply:
I did it to have the `summary` be the same for all 3 versions (WinForms, OffScreen, and WPF)

Reply:
Any objection to removing it? Whilst it's somewhat harmless I'd rather avoid using **new** unless there's a really compelling reason, I just look at it and think code smell, and puzzle why it's there. Maybe it's just me....

Reply:
Not at all, removed both the `Dispose` and `Finalizer` since their function is already taken care of by the base class

Reply:
Thanks 👍